src/SonarScanner.MSBuild.PreProcessor/WebServer/SonarCloudWebServer.cs

```diff
@@ -96,7 +95,7 @@ public override async Task<IList<SensorCacheEntry>> DownloadCache(ProcessedArgs
             }
             catch (Exception e)
             {
-                logger.LogWarning(Resources.WARN_IncrementalPRCacheEntryRetrieval_Error, e.Message);
+                logger.LogDebug(Resources.WARN_IncrementalPRCacheEntryRetrieval_Error, e.Message);
```

Comment:
Some context: Everything that used to `LogWarning` on exceptions is now a `LogDebug`, to prevent too much noise.

If a user does not have verbose logs enabled, they will see something like:

```
Cache data is empty. A full analysis will be performed.
```

...coming from the CacheProcessor as an `Info` log.

If they do have verbose logs enabled, they will see the `prepare_read` logs added below.